### PR TITLE
API specifications as per OpenAPI3.0 standard

### PR DESCRIPTION
**Pull request by muralidatadeli**

### CODE REVIEW

File: resources/aoiViewerApiOA3.yml [hunk 1]
```diff
@@ -0,0 +1,1965 @@
+openapi: 3.0.1
+info:
+  title: Geospatial Application
+  description: Application API.
+  version: '0.0'
+servers:
+  - url: https://api.test.datadeli.com/static/aoiViewerApi.yml
```
Comment (upendra-datadeli) [inline resources/aoiViewerApiOA3.yml:7]: I don't think url should be this.

File: resources/aoiViewerApiOA3.yml [hunk 2]
```diff
@@ -0,0 +1,1965 @@
+openapi: 3.0.1
+info:
+  title: Geospatial Application
```
Comment (upendra-datadeli) [inline resources/aoiViewerApiOA3.yml:3]: We need to come up with a better name

**Comment by upendra-datadeli** [comment]
Please resolve the conflicts

**Review by upendra-datadeli — CHANGES_REQUESTED** [review]

File: resources/aoiViewerApiOA3.yml (hunk 1 above)
Comment (muralidatadeli) [reply resources/aoiViewerApiOA3.yml]: corrected as per https://swagger.io/docs/specification/api-host-and-base-path/

File: resources/aoiViewerApiOA3.yml (hunk 2 above)
Comment (muralidatadeli) [reply resources/aoiViewerApiOA3.yml:3]: Ok. Please suggest

**Comment by upendra-datadeli** [comment]
Can you please make sure it works with AppUI before merging? Thanks!

**Review by upendra-datadeli — APPROVED** [review]

**Pull request merged**